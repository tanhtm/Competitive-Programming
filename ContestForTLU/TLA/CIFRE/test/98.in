2 445184823136
