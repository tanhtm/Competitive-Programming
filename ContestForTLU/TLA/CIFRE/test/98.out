-6464513987661639341